core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1344,13 +1351,16 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
   private case class ProducerBuilder() extends ClientBuilder[KafkaProducer[String, String]] {
     private var _retries = 0
     private var _acks = -1
+    private var _requestTimeoutMs = 30000L
```

Comment:
It looks like this is intended to be the default in `ProducerConfig`. Maybe we should refer to the value there?

Reply:
@ijuma Thanks for the review. `ProducerConfig` hard-codes the default in the config definition, so there doesn't seem to be an easy way to use that constant. So I will merge as-is for now. Let me know if I should add a re-usable constant in `ProducerConfig` for default values and I can do that in a follow-on PR since that is public API. Thanks.

Reply:
Yeah, it seems like an oversight that such constants only exist for some of the values. But it's OK to leave as is for now.